recipe/meta.yaml

```diff
@@ -94,11 +110,11 @@ build:
     - libcoi_device.so.0
     - /usr/lib/libstdc++.so.6       # [linux]
     - /usr/lib64/libstdc++.so.6     # [linux]
-    - /usr/lib/libc++.1.dylib       # [osx]
     # hooray, windows
     - "C:\\Windows\\System32\\WINTRUST.dll"
-
+{% endif %}
 outputs:
+{% if not osx %}
```

Comment:
my guess is that this have to be handled by line 98 ?

skip: True # [not x86 and not (linux64 or win)]

Reply:
skip in global build section will cause all outputs to cancel.